code/lib/util.js

```diff
@@ -122,30 +128,30 @@ function buildAcceptedAnswer(answer, options) {
 
   return {
     answer: acceptedAnswer,
-    alias: alias
-  }
+    alias: alias,
+  };
 }
 
 function buildQuizzes(searchTerm) {
-  var quizzes = require("../data/quizzes");
+  var quizzValue = quizzes;
   if (searchTerm) {
-    const foundQuiz = findItems(quizzes, searchTerm)
+    const foundQuiz = findItems(quizzes, searchTerm);
     if (foundQuiz.length > 0) {
-      quizzes = foundQuiz;
+      quizzValue = foundQuiz;
     }
   }
   var formattedQuizzes = [];
   //read the questions in the quiz and initialize the state
-  for (var i = 0; i < quizzes.length; i++) {
-    var quiz = quizzes[i];
+  for (var i = 0; i < quizzValue.length; i++) {
+    var quiz = quizzValue[i];
```

Comment:
Can you test and see if this works?

```suggestion
  const foundQuiz = searchTerm ? findItems(quizzes, searchTerm) : undefined;

  var formattedQuizzes = [];
  //read the questions in the quiz and initialize the state
  for (var i = 0; i < foundQuiz.length; i++) {
    var quiz = foundQuiz[i];
```

Reply:
with the suggested code the capsule only works when you specify the kind of quiz you want example:
"start quiz" doens't work
![image](https://user-images.githubusercontent.com/111146363/232051979-7f2de95f-f433-4c89-bf80-8e194d553452.png)
"give me a funny quiz" works
![image](https://user-images.githubusercontent.com/111146363/232052283-4b9db434-d023-4553-b637-01dbd164aa55.png)